data/json/overmap/overmap_special/specials.json

```diff
@@ -3558,6 +3558,15 @@
     "occurrences": [ 0, 30 ],
     "flags": [ "CLASSIC", "WILDERNESS" ]
   },
+  {
+    "type": "overmap_special",
+    "id": "Hot Springs",
+    "overmaps": [ { "point": [ 0, 0, 0 ], "overmap": "hot_springs_north" } ],
+    "locations": [ "field" ],
+    "city_distance": [ 1, -1 ],
+    "occurrences": [ 0, 30 ],
```

Comment:
Are hot springs that much rife in New England? Up to 30 locations per overmap feels like way too much for them.

Reply:
Yea they seem fairly rare in the region.

Reply:
Oh, this was supposed to be rarer, yeah that is way too much!
What would be a better occurance rate?

Reply:
I'm not sure, but something like up to 1 or 2 per overmap.

Reply:
Set it at 2, which should possibly make it rarer